site/content/en/docs/Integration Patterns/high-density-gameservers.md

```diff
@@ -7,7 +7,7 @@ description: >
   How to run multiple concurrent game sessions in a single GameServer process.
 ---
 
-{{< alpha title="Allocation State Filter" gate="StateAllocationFilter" >}}
+{{< beta title="Allocation State Filter" gate="StateAllocationFilter" >}}
```

Comment:
These changes will also need to be `feature` shortcoded, such that they only get updated when we do our next release.

Reply:
I added `feature` shortcode.